book-store/src/containers/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+
+/**
+ * Main Component
+ */
+class App extends React.Component {
+    constructor(){
+        super();
+    };
+
+    render(){
+        return (
+            <div>
```

Comment:
Why would you want to add routes here, you have already written routes in your index.js